tests/components/aseko_pool_live/test_config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Test the Aseko Pool Live config flow."""
+from unittest.mock import AsyncMock, patch
+
+from aioaseko import AccountInfo, APIUnavailable, InvalidAuthCredentials
+import pytest
+
+from homeassistant import config_entries, setup
+from homeassistant.components.aseko_pool_live.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import RESULT_TYPE_CREATE_ENTRY, RESULT_TYPE_FORM
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need to set up the persistent_notification integration anymore in the tests.